wiki/Front-end_with_curl.md

```diff
@@ -0,0 +1,284 @@
+# Curl Examples to front-end
+
+The front-end is accessable not only via the web GUI.
+
+<h3>Introduktion</h3>
+<p>
+The front-end has a bridge functionality that is built in as a part to the front-ends web server and handles request coming from the web GUI towards any of the back-ends configured.
+The front-end may also be used by other applications such as CURL or any kind of program that can make HTTP(S) requests.
+</p>
+
+<h3>The front-end handels:</h3>
+<ul>
+   <li>POST, PUT and GET requests</li>
+   <li>Default back-end</li>
+   <li>Input back-end</li>
+</ul>
+<p>
+While the web gui may use the back-end instances list and select different back-ends in an easy way, a user that uses the front-end without the included web GUI may need to specify a back-end URL.
```

Comment:
Once you write web gui and then web GUI. It needs to be more consistent.

Reply:
Done.